### ISSUE TITLE
[Toolkit feature]: Working delete button

### ISSUE THREAD

**Issue by PapaSlask** [NVIDIAGameWorks/rtx-remix#359]
### What is the context of the feature/improvement?

Would love for the delete button to work. Right now we have to press the small rubbish bins to delete selected items.

### Please describe the feature/improvement?

As described above

### What would be your solution? (optional)

...

### Version

2024.1.1

**Comment by nvdamien** [comment]
Hi,

Can you give us more detail please? Are you talking about the keyboard?
Do you have any screenshot?

**Comment by PappaSlask** [comment]
> Hi,
> 
> Can you give us more detail please? Are you talking about the keyboard? Do you have any screenshot?

More details? I mean, the delete key on the keyboard does nothing in Kit. It doesn't have the function it's suppose to have: Delete selected stuff. I don't know how I can be any more clear sir.

**Comment by nvdamien** [comment]
Hi @PappaSlask ,

The original message was:

> Would love for the delete button to work

We don't know if it was about a delete **button** in the app (a button that we click on it. If yes, which one?), or it was the DEL key from the keyboard.
The initial message doesn't say anything about keyboard, that why we are asking more details.

Filled as REMIX-2733

Thank you

**Comment by PappaSlask** [comment]
> Hi @PappaSlask ,
> 
> The original message was:
> 
> > Would love for the delete button to work
> 
> We don't know if it was about a delete **button** in the app (a button that we click on it. If yes, which one?), or it was the DEL key from the keyboard. The initial message doesn't say anything about keyboard, that is why we are asking more details.
> 
> Filled as REMIX-2733
> 
> Thank you

Right, Apologies for not being clear enough. I made the issue in a bit of a rush. I should take more time in describing issues.

**Comment by PappaSlask** [comment]
@nvdamien Is this still being considered sir? I was missing this feature the other day when I wanted to delete a bunch of lights I didn't need.

**Comment by nvdamien** [comment]
Hi @PappaSlask ,

Yes. We are working through the ticket backlog, we will let you know when there is an update.
